src/pyproject_toml.rs

```diff
@@ -86,6 +86,8 @@ impl GlobPattern {
 #[serde(rename_all = "kebab-case")]
 pub struct ToolMaturin {
     // maturin specific options
+    // extension module name, accepts setuptools style import name like `foo.bar`
+    name: Option<String>,
```

Comment:
I feel that `name` isn't a good name for this, how about `extension-name` / `extension-module`?

@konstin

Reply:
yeah, maybe `rust-module-name` or `native-module-name`? i think `extension` a bit misleading because in maturin's case it's not an extension, it's the main thing (and the metaphor of extending cpython by adding new built-in modules is lost on most users)

Reply:
I think `module-name` is a good alternative.